.travis.yml

```diff
@@ -1,13 +1,41 @@
 sudo: false
+
 language: python
+
 cache: pip 
+
 python:
   - 3.5
+
+env:
+  - PYDATA=true
+  - PYDATA=false CHECK_MIGRATION=true
+
+before_install:
+  - git config --global user.email "you@example.com"
+  - git config --global user.name "Your Name"
+  - if [[ $TRAVIS_PULL_REQUEST != false ]]; then
+      git pull -q --no-edit origin develop;
+    fi;
+  - if [[ $PYDATA == true ]]; then
+      git pull -q --no-edit origin deploy-pydata;
```

Comment:
What exactly is in `deploy-pydata`?

Are you sure `git pull` is the right way to go?

Can we use `git config --local` instead of `git config --global`?